frontend/src/components/RoadTripTopBar/RoadTripTopBar.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Button, AppBar, Toolbar, withStyles, makeStyles } from '@material-ui/core';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import GroupAddRoundedIcon from '@material-ui/icons/GroupAddRounded';
+import HomeRoundedIcon from '@material-ui/icons/HomeRounded';
+import { NavLink, useParams } from 'react-router-dom';
+import SideBar from '../SideBar/SideBar';
+import styles from './RoadTripTopBar.module.css';
+import useGet from '../../hooks/useGet';
+import AttendeesModal from './AttendeesModal/AttendeesModal';
+// import * as PropTypes from "prop-types";
+
+// TODO Flags for host
+// class FlagsProvider extends React.Component {
+//   render() {
+//     return null;
+//   }
+// }
+//
+// FlagsProvider.propTypes = {
+//   features: PropTypes.shape({ moderate: PropTypes.bool }),
+//   children: PropTypes.node
+// };
+const useStyles = makeStyles(() => ({
+  grow: {
+    flexGrow: 1,
+  },
+}));
+
+const IconButton = withStyles({
+  label: {
+    color: '#24305e',
+    '&:hover': {
+      color: '#374785',
+    },
+  },
+})(Button);
+
+const CustomTopBar = withStyles({
+  root: {
+    backgroundColor: 'white',
+  },
+})(AppBar);
+
+export default function RoadTripTopBar() {
+  const { id } = useParams();
+  const { response } = useGet(`/api/roadtrip/${id}`);
+  const classes = useStyles();
+
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const handleOpenModal = () => {
+    setModalOpen(true);
+  };
+
+  const handleCloseModal = () => {
+    setModalOpen(false);
+  };
+
+  return (
+    <CustomTopBar position="static">
+      <Toolbar variant="dense">
+        <SideBar />
+        <p className={styles.title}>{response?.data.name}</p>
+        <div className={classes.grow} />
+        <IconButton>
+          <GroupAddRoundedIcon onClick={handleOpenModal} />
```

Comment:
is it possible to have this icon be one without a "+" when you are not an organiser? e.g. `import GroupIcon from '@material-ui/icons/Group';` instead

Reply:
Ahh good catch, I'll include this now